strax/context.py

```diff
@@ -1361,6 +1358,106 @@ def _apply_function(self, data, targets):
             data = function(data, targets)
         return data
 
+    def _copy_to_frontend(self, run_id, target,
```

Comment:
Perhaps this should not be `_`-ed. If the reviewer has an opinion I'm interested

Reply:
On the one hand I agree with you that this function should just be used by a very limited amount of people who know what they are doing and hence make the function not "visible". On the other hand I think it is very unlikely that anyone will use this function except for the experts. My/our concerns are very limited, and the underscore usually indicates that the function is a private member, I agree with your statement to remove it.